.bazelignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+dist
+bazel-out
```

Comment:
Surprisingly this is not already ignored?

Reply:
yaeh